### DISCUSSION TITLE
Reason for files start showing reduced coverage?

### DISCUSSION THREAD

**Discussion by elenajdanova** [ArtiomTr/jest-coverage-report-action#264] [Q&A]
I was introducing minor changes in my code, additionally updating the tests.
I didn't add any tests, I didn't delete any tests.
Output is giving me drastically reduced coverage for several files, one of them wasn't touched at all in mentioned PR. Can somebody explain what's going on?

![Screen Shot 2022-04-27 at 11 50 29 AM](https://user-images.githubusercontent.com/26873786/165560535-a11c6804-e2a7-44b9-8794-7395cdffb8d1.png)

https://user-images.githubusercontent.com/16534638/162859122-be977572-5bef-4d46-92ff-c68d26f340e9.png

**Comment by ArtiomTr** [comment]
Answered in https://github.com/ArtiomTr/jest-coverage-report-action/issues/265#issuecomment-1111462805
